system-parachains/collectives/collectives-polkadot/src/fellowship/mod.rs

```diff
@@ -244,3 +251,120 @@ impl pallet_salary::Config<FellowshipSalaryInstance> for Runtime {
 	// Total monthly salary budget.
 	type Budget = ConstU128<{ 100_000 * USDT_UNITS }>;
 }
+
+parameter_types! {
+	// TODO: reference the constant value from common crate when polkadot-sdk 1.5 is released.
+	pub const FellowshipTreasuryPalletId: PalletId = PalletId(*b"py/feltr");
+	pub const HundredPercent: Permill = Permill::from_percent(100);
+	pub const Burn: Permill = Permill::from_percent(0);
+	pub const MaxBalance: Balance = Balance::max_value();
+	// The asset's interior location for the paying account. This is the Fellowship Treasury
+	// pallet instance (which sits at index 65).
+	pub FellowshipTreasuryInteriorLocation: InteriorMultiLocation = PalletInstance(65).into();
+}
+
+/// [`PayOverXcm`] setup to pay the Fellowship Treasury.
+pub type FellowshipTreasuryPaymaster = PayOverXcm<
+	FellowshipTreasuryInteriorLocation,
+	crate::xcm_config::XcmRouter,
+	crate::PolkadotXcm,
+	ConstU32<{ 6 * HOURS }>,
+	VersionedMultiLocation,
+	VersionedLocatableAsset,
+	LocatableAssetConverter,
+	VersionedMultiLocationConverter,
+>;
+
+pub type FellowshipTreasuryInstance = pallet_treasury::Instance1;
+
+impl pallet_treasury::Config<FellowshipTreasuryInstance> for Runtime {
+	// The creation of proposals via the treasury pallet is deprecated and should not be utilized.
+	// Instead, public or fellowship referenda should be used to propose and command the treasury
+	// spend or spend_local dispatchables. The parameters below have been configured accordingly to
+	// discourage its use.
+	// TODO: replace with `NeverEnsure` once polkadot-sdk 1.5 is released.
```

Comment:
we should have issues linking to todos to remind people there are future work to be done